modules/social_features/social_embed/social_embed.install

```diff
@@ -127,3 +127,29 @@ function social_embed_uninstall() :void {
 function social_embed_update_last_removed() : int {
   return 11005;
 }
+
+/**
+ * Change lazy-filter to use the social lazy-filter.
+ */
+function social_embed_update_124000(): void {
+  $config_factory = \Drupal::configFactory();
+  $text_formats = [
+    'filter.format.full_html',
+    'filter.format.basic_html',
+  ];
+
+  foreach ($text_formats as $text_format) {
+    $configuration = $config_factory->getEditable($text_format);
+    $filters = $configuration->get('filters');
+
+    if (isset($filters['lazy_filter'])) {
+      $filters['social_embed_lazy_filter'] = $filters['lazy_filter'];
+      $filters['social_embed_lazy_filter']['id'] = 'social_embed_lazy_filter';
+      $filters['social_embed_lazy_filter']['provider'] = 'social_embed';
+      unset($filters['lazy_filter']);
```

Comment:
It seems the `'lazy_filter'` is not always set, which means in that case it won't run. 

I'm not sure if we really need the same filter, the previous settings we're very much coming from the lazy module, but ours should be much easier.

Especially since it's progressively enhanced, and we don't have any settings form for a SM to change it, I would just make sure we add our own filter to it anyway, without Settings, just for Iframes.